plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/data/AttributeMetaDataProxy.java

```diff
@@ -81,6 +81,11 @@ public DBSObject getSource() {
     @NotNull
     public String getLabel()
     {
+        DBSAttributeBase proxyAttribute = getProxyAttribute();
+        if (proxyAttribute instanceof DBCAttributeMetaData) {
+            return ((DBCAttributeMetaData) proxyAttribute).getLabel();

```

Comment:
Just a nitpick: since we use Java 17, you can do the following:

```suggestion
        if (proxyAttribute instanceof DBCAttributeMetaData attributeMetaData) {
            return attributeMetaData.getLabel();

```

Reply:
Addressed in the recent version. Also applied on the code that I used for the fix. @ShadelessFox Resolve at will.